readme.md

```diff
@@ -38,10 +38,17 @@ The source object to copy properties from.
 
 #### mapper
 
-Type: `(sourceKey, sourceValue, source) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
+Type: `(sourceKey, sourceValue, source, path) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
 
 A mapping function.
 
+##### path
+
+Type: `string[]|undefined`\
+
+If `deep === true`, this is the sequence of keys to reach the current value from the `source`;
+otherwise it is `undefined`.
```

Comment:
I would explicitly mention how array paths are represented. It may not be obvious to the user.

Reply:
Not everyone knows that arrays are just objects with numbered keys. It doesn't hurt to be clear about it with a more user-friendly text and maybe an example.